ocs_ci/ocs/resources/objectbucket.py

```
@@ -292,7 +292,9 @@ def verify_deletion(self, timeout=60, interval=5):
                     logger.info(f"{self.name} still exists. Retrying...")
         except TimeoutExpiredError:
             logger.error(f"{self.name} was not deleted within {timeout} seconds.")
```

Ruff (0.15.13)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)